my-app/src/components/utilities/utilities.js

```diff
@@ -0,0 +1,179 @@
+import react from "react";
+import { useEffect, usestate, useReducer } from "react";
+import axios from "axios";
+import { useProducts } from "../../contexts/ProductContext";
+
+export const postRequestCart = (prd) => {
+   (async function () {
+      try {
+         const response = await axios.post(
+            "https://myfeline-restapi.sohamparab13.repl.co/cart",
+            {
+               name: prd.name,
+               description: prd.description,
+               images: prd.images,
+               price: prd.price,
+               rating: prd.rating,
+               total_ratings: prd.total_ratings,
+               category: prd.category,
+               brand: prd.brand,
+               quantity: prd.quantity,
+            }
+         );
+         console.log(response, "adawda");
+      } catch (error) {
+         console.log(error);
+      }
+   })();
+};
+
+export const postRequestWishlist = (prd) => {
+   (async function () {
+      try {
+         const response = await axios.post(
+            "https://myfeline-restapi.sohamparab13.repl.co/wishlist",
+            {
+               name: prd.name,
+               description: prd.description,
+               images: prd.images,
+               price: prd.price,
+               rating: prd.rating,
+               total_ratings: prd.total_ratings,
+               category: prd.category,
+               brand: prd.brand,
+               quantity: prd.quantity,
```

Comment:
There's another way to do this, if you are using MongoDB you can do this by referencing `products` in your cart schema using the `item._id`. LMK if you need help with this.